src/seedu/addressbook/data/person/Address.java

```diff
@@ -25,7 +29,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(address)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
-        this.value = address;
+        String[] array = address.split(",\\s");
```

Comment:
You need to consider what if the user doesn't put a space after a comma. Try to come up with a more robust approach.